.github/workflows/npm.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow runs a test build for npm against changes on main or PRs
+
+name: Npm Build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: macos-latest
+    
+    steps:
+      - name: Actions checkout
+        uses: actions/checkout@v3
+      
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: latest
+          registry-url: https://registry.npmjs.org/
+          
+      - name: Setup Deno
+        uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
+
+      - name: Run build_npm.ts
+        run: deno run -A build_npm.ts
```

Comment:
Running this without another argument, the `Deno.args[0]` we pass in the script to the `version` field becomes undefined and isn't emitted in the package.json. Not sure if that's relevant.

Reply:
Yeah true - I figured since the purpose of this workflow step is just to ensure that the build completes, we don't have to populate the version.